src/plugins/bfetch/server/streaming/create_streaming_response_stream.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Stream, PassThrough } from 'stream';
+import { StreamingResponseHandler } from '../../common/types';
+
+const delimiter = '\n';
```

Comment:
You called this `splitter` in the client. I think delimiter is a more standard name, so maybe we should use the same name on the client.

Reply:
First, I actually called it `delimiter` but then I saw in autocompletion suggestion `String.prototype.split` names it `splitter`, so for consistency I named it `splitter`, too. But I can call it `delimiter`.

Reply:
Renamed to `delimiter` in client code.